docs/cloud/cloud-troubleshooting/issue-cloud-version-unknown.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Database "Unknown"
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Why is my database's version "Unknown"?
```

Comment:
So here we need slightly different headings.

An example of an issue is here:
https://github.com/FeatureBaseDB/featurebase-docs/blob/main/docs/cloud/cloud-troubleshooting/issue-int-limitations.md

Three headings:

```
# Issue - unknown database version

{% include /cloud-db/cloud-summary-db-version.md %}

An unknown database version can occur for a number of reasons

## Causes

A database has an **unknown** version when it fails to respond to UI queries.

The causes may be any of the following:

| Description | Database status |
|---|---|
| Database is not yet created | CREATING<br/>PROVISIONING |
| Database deletion in progress | DELETED<br/>DEPROVISIONING |
| Database has not started properly | FAILED |

## Solution

<Need some solutions or workarounds here>

## Further information

* [Learn about the database status](/docs/cloud/cloud-databases/cloud-db-states)
```